examples/go.mod

```diff
@@ -1,10 +1,15 @@
 module github.com/pulumi/pulumi-aws-native/examples
 
-go 1.14
+go 1.16
+
+replace github.com/pulumi/pulumi-aws-native/sdk => ../sdk
```

Comment:
Interesting! Will this pick up the dev build of the provider? How does this interact with the `Dependencies` option of `ProgramTestOptions` ?

Reply:
I chatted with @stack72 about this and apparently was doing this completely wrong. Updated and should hopefully be working now.